IronSoftware.Drawing/IronSoftware.Drawing.Common/AnyBitmap.cs

```diff
@@ -450,6 +466,131 @@ public static AnyBitmap FromUri(Uri Uri)
             }
         }
 
+        /// <summary>
+        /// Gets colors depth, in number of bits per pixel.
+        /// </summary>
+        public int BitsPerPixel
+        {
+            get
+            {
+                return Image.PixelType.BitsPerPixel;
+            }
+        }
+
+        /// <summary>
+        /// Returns the number of frames in our loaded Image.  Each “frame” is a page of an image such as  Tiff or Gif.  All other image formats return 1. 
+        /// </summary>
+        /// <seealso cref="GetAllFrames" />
+        public int FrameCount
+        {
+            get
+            {
+                return Image.Frames.Count;
+            }
+        }
+
+        /// <summary>
+        /// Returns all of the cloned frames in our loaded Image. Each "frame" is a page of an image such as Tiff or Gif. All other image formats return an IEnumerable of length 1.
+        /// </summary>
+        /// <seealso cref="FrameCount" />
+        /// <seealso cref="System.Linq" />
+        public IEnumerable<AnyBitmap> GetAllFrames
+        {
+            get
+            {
+                if (FrameCount > 1)
+                {
+                    List<AnyBitmap> images = new List<AnyBitmap>();
+
+                    for (int currFrameIndex = 0; currFrameIndex < FrameCount; currFrameIndex++)
+                    {
+                        images.Add(Image.Frames.CloneFrame(currFrameIndex));
+                    }
+                    return images;
+                }
+                else
+                {
+                    return new List<AnyBitmap>() { this.Clone() };
+                }
+            }
+        }
+
+        /// <summary>
+        /// Creates a multi-frame TIFF image from multiple AnyBitmaps.
+        /// <para>All images should have the same dimension.</para>
+        /// <para>If not dimension will be scaling to the largest width and height.</para>
+        /// <para>The image dimension still the same with original dimension with black background.</para>
+        /// </summary>
+        /// <param name="imagePaths">Array of fully qualified file path to merge into Tiff image.</param>
+        /// <returns></returns>
+        public static AnyBitmap CreateMultiFrameTiff(IEnumerable<string> imagePaths)
```

Comment:
Are we using `Create` for all of these loading methods. I would personally use `AnyBitmap.LoadMultiFrameTiff(`

Reply:
Or: `LoadFromMultiFrameTiff`

Reply:
This function will merge all images into 1 Tiff image.
So, I think Create is make sense for this function because it's not loaded an image.